test/volumeindicatoradapter/DefaultVolumeIndicatorAdapter.test.ts

```diff
@@ -119,6 +119,53 @@ describe('DefaultVolumeIndicatorAdapter', () => {
       expect(volumeUpdate).to.equal(2);
       expect(attendeeIdUpdate).to.equal(2);
     });
+
+    it('handles out of order stream ids', () => {
+      const streamInfo1Join = SdkAudioStreamIdInfo.create();
+      const streamInfo1Frame = SdkAudioStreamIdInfoFrame.create();
+      const streamInfo2Join = SdkAudioStreamIdInfo.create();
+      const streamInfo2Leave = SdkAudioStreamIdInfo.create();
+      const streamInfo2Frame = SdkAudioStreamIdInfoFrame.create();
+      streamInfo1Join.audioStreamId = 1;
+      streamInfo1Join.attendeeId = fooAttendee;
+      streamInfo1Join.externalUserId = fooExternal;
+      streamInfo1Frame.streams = [streamInfo1Join];
+      streamInfo2Join.audioStreamId = 2;
+      streamInfo2Join.attendeeId = fooAttendee;
+      streamInfo2Join.externalUserId = fooExternal;
+      streamInfo2Leave.audioStreamId = 1;
```

Comment:
Minor: For the leave frame, do we only send audioStreamId without attendee Id and externalUserId? If not, could we set it here so it is easy to understand that this is the same user from streamInfo1.

Reply:
Leave frames are indicated by the absence of an attendee ID and external user ID, so that is intentional.